docker/README.md

```diff
@@ -0,0 +1,101 @@
+**This documentation focuses on the Deployer Docker image and its usage, not the Deployer itself. If you want learn more about the Deployer, please read [its documentation](../packages/deployer/README.md).**
+
+# Build
+You can build the Docker image by running:
+```bash
+docker build -f docker/Dockerfile -t api3/deployer:latest .
+```
+
+Before the build, all Deployer dependencies (but not the Deployer itself) must be available in the package registry.
+
+> You can use build argument `yarnRegistry` to specify a different package registry for package installation. This is useful for testing a Docker build with unreleased dependencies. You can use for example [Verdaccio](https://verdaccio.org/) for this purpose.
+
+# Configuration
+## Credentials
+In order to deploy Airnode to a cloud provider like AWS, you need to provide your cloud credentials to the container. Airnode currently only supports deploying to [AWS](https://aws.amazon.com/lambda/).
+
+### AWS
+There are two options to provide your AWS credentials to the Docker container
+
+#### Option A: Environment variables:
+Create a `aws.env` file with following content and reference it when running Docker
+```
+AWS_ACCESS_KEY_ID=...
+AWS_SECRET_ACCESS_KEY=...
+AWS_SESSION_TOKEN=...
+```
+
+```bash
+$ docker run --env-file aws.env ...
+```
+
+Alternatively, you can provide the environment variables in separately to the Docker command
+```bash
+docker run -e AWS_ACCESS_KEY_ID=... -e AWS_SECRET_ACCESS_KEY=... ...
+```
+
+#### Option B: A configuration file:
+You can use a [credential file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where) to store toy AWS credentials. Passing this file to the container will allow the Deployer to use these credentials:
+```bash
+docker run -v ${HOME}/.aws/credentials:/root/.aws/credentials ...
+```
+
+## Permissions
+The Deployer is by default run by the user `root`. This may lead to some permission issues since the Deployer provides an output in a form of a `receipt.json` file. To avoid the permission problems you can specify with what [UID (user identifier)](https://en.wikipedia.org/wiki/User_identifier) and [GID (group identifier)](https://en.wikipedia.org/wiki/Group_identifier) should be the Deployer run. You can do that by setting the environment variables `USER_ID` and `GROUP_ID`:
```

Comment:
```suggestion
By default, the Deployer is run by the user `root`. This may lead to some permission issues since the Deployer provides an output in a form of a `receipt.json` file. To avoid any permission problems, you can specify the [UID (user identifier)](https://en.wikipedia.org/wiki/User_identifier) and [GID (group identifier)](https://en.wikipedia.org/wiki/Group_identifier) that the Deployer should use. You can do that by setting the environment variables `USER_ID` and `GROUP_ID`:
```